README.md

```diff
@@ -0,0 +1,56 @@
+## Dispatch Event Driver for AWS
+This repo contains a template for aws SQS event driver support for Dispatch.
+
+### 1. Get AWS credentials
+Follow [`Get AWS Access Key`](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/setting-up.html) to obtain aws credential. Modify `<ACCESS_KEY_ID>` and `<SECRET_ACCESS_KEY>` in **credentials** file under this repo:
+```
+[default]
+aws_access_key_id = <ACCESS_KEY_ID>
+aws_secret_access_key = <SECRET_ACCESS_KEY>
+```
+
+### 2. Create AWS SQS Queue *[will be managed by Dispatch]*
+Create a standard AWS SQS with name *dispatch*, and also enable `long-polling` by setting `Receive Message Wait Time` attribute to 20.
+
+For more details, refer to [SQS long-polling](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-long-polling.html)
+
+
+### 3. Create Rules in AWS CloudWatch *[will be managed by Dispatch]*
+In *AWS console* -> *CloudWatch* -> *Event* -> *Rules* -> *Create Rules* for the event that want to be received by Dispatch. Choose *AWS Console Sign-in* -> *Sign-in Events* for example
+
+For Event Source, take a note of the `source` field of the chosen event, this will be used in creation of Dispatch subscription later.
+> **Note:** Most aws events display `source` field under *Event Patter Preview* section, if not, please expand *Show sample events* and look for `source` field.
+
+For Event Targets, choose the SQS queue *dispatch* created in step 2.
+
+
+### 4. Build Event Driver Image
+```bash
+$ docker build -t dispatch-events-aws .
+```
+
+### 5. Create Eventdrivertype in Dispatch
+```bash
+$ dispatch create eventdrivertype aws-event-driver-type dispatch-events-aws:latest
+Created event driver type: aws-event-driver-type
```

Comment:
I would use `aws-sns` or something similar as a name here. since you are creating an event driver type, the `event-driver-type` suffix is not required (it's implied).